language-server/modules/langserver-compiler/src/test/java/org/ballerinalang/langserver/compiler/workspace/WorkspaceDocumentTest.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.ballerinalang.langserver.compiler.workspace;
+
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+/**
+ * Tests for WorkspaceDocument.
```

Comment:
Let's add the version

Reply:
Fixed in [ecaf52d](https://github.com/ballerina-platform/ballerina-lang/pull/10194/commits/ecaf52d98daa0a19f68c882272fbae7284876b24)